src/pages/ImportData.js

```diff
@@ -118,10 +135,40 @@ const ImportData = () => {
         notify('success', 'Data imported successfully.', { timeout: 3000 })
       }],
       ['snapshot', async () => {
-        snapshots && snapshots.length > 0 ?
-          await Promise.allSettled(_.map(({ title, id }) => Ajax().Workspaces.workspace(namespace, name).importSnapshot(id, title), snapshots)) :
+        if (snapshots?.length > 0) {
+          const responses = await Promise.allSettled(
+            _.map(({ title, id, description }) => {
+              // Normalize the title:
+              // Importing snapshot will throw an "enum" error if the title has any spaces or special characters
+              // Replace all whitespace characters with _
+              // Then replace all non alphanumeric characters with nothing
+              const normalizedTitle = _.flow(
+                _.replace(/\s/g, '_'),
+                _.replace(/[^A-Za-z0-9\s-_]/g, '')
+              )(title)
```

Comment:
What if:
```suggestion
              const normalizedTitle = _.replace(/[^\w-]/g, char => /\s/.test(char) ? '_' : '', title)
```

Reply:
I think this is a lot harder to read, going to vote to leave it as the flow! Any other opinions?

Reply:
If you could remove `\s` from the second regex, since it should be gone by then, that would work for me.

Reply:
done